src/components/UI/Dropdown/Dropdown.tsx

```diff
@@ -10,10 +13,13 @@ export const Dropdown: FC<MultiDropdownProps> = ({
   disabled,
   className,
 }) => {
-  // console.warn("Dropdown render")
   const [isVisible, setIsVisible] = useState(false);
 
-  const itemOnClick = (el: Options) => {
+  const toggleList = () => {
+    setIsVisible(!isVisible);
```

Comment:
в сеттер состояния можно передавать не значение, а коллбэк, где аргумент – предыдущее значение состояния, а возвращаемое значение – новое состояние
```suggestion
    setIsVisible((prevIsVisible) => !prevIsVisible);
```
это полезно, когда следующее состояние зависит от предыдущего (как в данном случае)